test/functional/viml/errorlist_spec.lua

```diff
@@ -26,17 +26,11 @@ describe('setqflist()', function()
   it('sets w:quickfix_title', function()
     setqflist({''}, 'r', 'foo')
     command('copen')
-    eq(':foo', get_cur_win_var('quickfix_title'))
+    eq('foo', get_cur_win_var('quickfix_title'))
     setqflist({''}, 'r', {['title'] = 'qf_title'})
     eq('qf_title', get_cur_win_var('quickfix_title'))
   end)
 
-  it('allows string {what} for backwards compatibility', function()
-    setqflist({}, 'r', '5')
-    command('copen')
-    eq(':5', get_cur_win_var('quickfix_title'))
-  end)
```

Comment:
I guess this needs to be mentioned in following-head wiki.

Reply:
Good catch. That used to be the case at one point, but then more Vim patches were added and now it works again.

I put the test back.